cli/src/build.rs

```diff
@@ -138,20 +149,50 @@ pub fn build_project(arg_mode: &Option<String>, arg_platforms: &Option<Vec<Strin
     // Architecture selection for iOS or Android
     let selected_architectures = platform.select_archs();
 
+    // Noir only supports `aarch64-apple-ios` and `aarch64-linux-android`
```

Comment:
can we just select or show the possible architectures in the CLI?
e.g. 
if `Adapter` is `Noir`,
Then 
```sh
? Select ios architecture(s) to compile ›
✔ aarch64-apple-ios
```
or
```sh
✔ Select platform(s) to build for (multiple selection with space) · android
? Select android architecture(s) to compile (default: aarch64-linux-android) ›
⬚ x86_64-linux-android
⬚ i686-linux-android
⬚ armv7-linux-androideabi
✔ aarch64-linux-android
```

Reply:
Sure, I was thinking similar solutions like that, but it would take longer time to finish this. I thought this PR should be a quick one for NoirHack, but it works for me! If we does this for Noir, we should have the same for other adapters, it's more complicated.

Reply:
Do I need to fix it in this PR? or in the later one?

Reply:
we can fix it in later PR.
and we can do with this https://github.com/zkmopro/mopro/issues/294